content/en/blog/2026-04-08-cozystack-oss-health-section/index.md

```
@@ -5,6 +5,7 @@ date: 2026-04-08
 author: "Timur Tukaev"
 description: "We have launched a new OSS health section on the Cozystack website, with project stats refreshed automatically every month."
 ---
+### Cozystack have launched a new OSS health section on a website
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)